README.md

```diff
@@ -82,4 +82,37 @@ Accengage.trackLead(label, value);
 * @param object
 */
 Accengage.updateDeviceInfo(object);
+
+/**
+* Get Inbox Messages
+* Returns an array of Messages.
```

Comment:
Could you add the format of the message here or somewhere else where it makes sense?